### ISSUE TITLE
Plan to release v1.2.2?

### ISSUE THREAD

**Issue by YuukiARIA** [unity3d-jp/UnityChanSpringBone#23]
We are waiting for v1.2.2, especially including #12 and #10.

Is this repository still alive?

**Comment by Thaina** [comment]
I don't think this repo is alive anymore. And if there is any update for unitychan package in the next version related to bone system, it would just be utilizing Animation.Rigging instead

And also I speculate that we actually don't need to have unitypackage released. This repo itself, and any forked, can be installed with UPM directly with git url and I think it was a much better choice since it would decrease a bunch of folder populated in our project

I don't know the reason this repo owner was revert 1.2.2 for the latest commit but we can also specified commit hash in this url to UPM for the 1.2.2 like this`https://github.com/unity3d-jp/UnityChanSpringBone.git#0da293c6d55857f282c193be5e6948004b061543`
